### ISSUE TITLE
Question on improving face verification result

### ISSUE THREAD

**Issue by zul96a** [serengil/deepface#243]
Hello again sir!

Currently, I'm at the end of my face verification project. I'm looking for ways on how I can further improve
the face verification accuracy. 

I have a plan that where I want to apply transfer learning to Facenet by including my own custom dataset. This custom
dataset has very good amount of local faces (South East Asian) and there are at least 10000 pictures with some are of the same person. 

My questions are:
1. Does applying transfer learning would help increase the accuracy for face verification (if the two images are the same person)?
2. If so, is there any step that I can follow so I can apply transfer learning for the existing Facenet model? (like folder structures, etc etc)?

In any case, I would like thank you again sir as your repo really helps me developed a fantastic project so far!

**Comment by serengil** [comment]
Hello,

Thank you for your feedback. You might want to share the demo of your project.

Facenet model was trained by David Sandberg. Here, you can find its repo: https://github.com/davidsandberg/facenet . Your data set might be smaller than the reference study.
 
If you have succeeded to pass its accuracy, then go to HOME_FOLDER/.deepface/weights folder and replace facenet_weights.h5.

If your model would change, then you need to change the model structure here: https://github.com/serengil/deepface/blob/master/deepface/basemodels/Facenet.py

You can alternatively create a facenet_v2 model but this requires a lot of changes in the deepface repo.

**Comment by zul96a** [comment]
Thank you for your quick reply! My project are actually divided into smaller module where I handle the module of doing face verification between the user selfie and user Identification Card (the printed face on it).

I actually plotted ROC graph and calculated the AUC score for it to be around 0.84 which is respectable. Now, I'm looking ways on how to improve the score and accuracy if possible.

But since you mentioned my dataset might be too small, I'll have to think some other way.

By the way for the last part of your reply, is it possible if I use this repo's model structure (and the facenet_weights.h5) as my base model for transfer learning? And the last sentence in your reply seems indicate there might be a lot changes needed to be done if I changed the facenet model :0
